internal/pkg/service/stream/mapping/recordctx/fasthttp.go

```diff
@@ -0,0 +1,145 @@
+package recordctx
+
+import (
```

Comment:
recordctx provides information about such as: headers, body, ...
- It is lazy.
- It is thread/goroutine free.